apps/aecore/lib/aecore/account/account.ex

```diff
@@ -335,4 +347,55 @@ defmodule Aecore.Account.Account do
   def base58c_decode(bin) do
     {:error, "#{__MODULE__}: Wrong data: #{inspect(bin)}"}
   end
+
+  @spec rlp_encode(Account.t()) :: binary() | {:error, String.t()}
+  def rlp_encode(%Account{} = account) do
+    [
+      type_to_tag(Account),
+      get_version(Account),
+      account.pubkey,
+      account.nonce,
+      account.last_updated,
+      account.balance
+    ]
+    |> ExRLP.encode()
+  end
+
+  def rlp_encode(_) do
+    {:error, "Invalid Account structure"}
+  end
+
+  @spec rlp_decode(binary()) :: {:ok, Account.t()} | {:error, String.t()}
+  def rlp_decode(values) when is_binary(values) do
+    [tag_bin, ver_bin | rest_data] = ExRLP.decode(values)
+    tag = Serialization.transform_item(tag_bin, :int)
+    _ver = Serialization.transform_item(ver_bin, :int)
+
+    case tag_to_type(tag) do
+      Account ->
+        [pkey, nonce, height, balance] = rest_data
+
+        {:ok,
+         %Account{
+           pubkey: pkey,
+           balance: Serialization.transform_item(balance, :int),
+           last_updated: Serialization.transform_item(height, :int),
+           nonce: Serialization.transform_item(nonce, :int)
+         }}
+
+      _ ->
+        {:error, "Illegal Account serialization"}
```

Comment:
Maybe we should add a more descriptive error here: "Invalid tag in Account serialization". Also we started previously adding `__MODULE__` to every error. Maybe we should add it to all of the new errors also.